rdkshell.cpp

```diff
@@ -246,10 +269,25 @@ namespace RdkShell
         gRdkShellIsRunning = true;
         while( gRdkShellIsRunning )
         {
+            RdkShell::CompositorController::update();
+            uint32_t width = 0;
+            uint32_t height = 0;
+            RdkShell::EssosInstance::instance()->resolution(width, height);
+            glViewport( 0, 0, width, height );
+            glClearColor(0.0f, 0.0f, 0.0f, 0.0f);
+            glClear(GL_COLOR_BUFFER_BIT);
+
             const double maxSleepTime = (1000 / gCurrentFramerate) * 1000;
             double startFrameTime = microseconds();
             RdkShell::CompositorController::draw();
             RdkShell::EssosInstance::instance()->update();
+            #ifdef RDKSHELL_ENABLE_IPC
+            if (gIpcEnabled)
+            {
+                gServerMessageHandler->poll();
+            }
```

Comment:
This is a good place to read IPC messages when running RDKShell as a standalone client.  However, where are we reading the IPC messages from when running RDKShell as a thunder plugin?  Shouldn't that be in `RdkShell::update()`?